app/helpers/comments_helper.rb

```diff
@@ -361,4 +364,8 @@ def comments_are_moderated(commentable)
     parent = find_parent(commentable)
     parent.respond_to?(:moderated_commenting_enabled) && parent.moderated_commenting_enabled?
   end
+
+  def on_js_less_comment_form(commentable)
```

Comment:
I think there are scenarios where this will be the case when JS is enabled, for example using the reply link from an email (or opening in a new tab like the issue mentions). Perhaps something like `focused_on_comment` would be a better description of what this represents?